vmdk_plugin/vmdkops/esx_vmdkcmd.go

```diff
@@ -85,19 +86,30 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 	defer C.free(unsafe.Pointer(ans))
 
 	// connect, send command, get reply, disconnect - all in one shot
-	_, err = C.Vmci_GetReply(C.int(EsxPort), cmdS, beS, ans)
-
-	if err != nil {
-		var errno syscall.Errno
-		errno = err.(syscall.Errno)
-		msg := fmt.Sprintf("'%s' failed: %v (errno=%d).", cmd, err, int(errno))
-		if errno == syscall.ECONNRESET {
-			msg += " Hit communication issue with ESX (vmci or ESX service)\n"
-			msg += " Please refer to the FAQ https://github.com/vmware/docker-volume-vsphere/wiki#faq"
+	retryCount := 10
+	for {
+		retryCount--
+		_, err = C.Vmci_GetReply(C.int(EsxPort), cmdS, beS, ans)
+		if err != nil {
+			var errno syscall.Errno
+			errno = err.(syscall.Errno)
+			msg := fmt.Sprintf("'%s' failed: %v (errno=%d).", cmd, err, int(errno))
+			if retryCount == 0 {
+				if errno == syscall.ECONNRESET {
+					msg += " Hit communication issue with ESX (vmci or ESX service)\n"
```

Comment:
I think it's better to say "Communication issue...." instead of "hit communication issue"